spec/clean/clean.rb

```diff
@@ -0,0 +1,62 @@
+describe Daru::Clean do
+  before(:each) do
+    @df = Daru::DataFrame.new(
+      {
+        a: [1,2,3,4,5],
+        b: ['mal','femal','M','F','Fem'],
+        c: [10,nil,30,40,nil],
+        d: ['a,c','a','b','b','b,a'],
+        e: [:A,:N,:B,:C,:N]
+      })
+  end
```

Comment:
Please use "new" style of specs (see daru-io for good examples) -- with `subject`, `let` and so on.